agent/config/config_unix.go

```diff
@@ -44,7 +46,7 @@ const (
 func DefaultConfig() Config {
 	return Config{
 		DockerEndpoint:                     "unix:///var/run/docker.sock",
-		ReservedPorts:                      []uint16{SSHPort, DockerReservedPort, DockerReservedSSLPort, AgentIntrospectionPort, AgentCredentialsPort},
+		ReservedPorts:                      []uint16{SSHPort, DockerReservedPort, DockerReservedSSLPort, AgentIntrospectionPort, AgentCredentialsPort, AgentPrometheusExpositionPort},
```

Comment:
this port should probably also be opted in, otherwise it seems that an in-place agent upgrade might break customer's workflow if his task uses this port

Reply:
Removed adding the port here and moved it to PlatformOverrides